AllJoyn/Platform/Bridge/BridgeNET/BridgeNET.vcxproj

```diff
@@ -211,7 +212,9 @@
     <ClInclude Include="Stdafx.h" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="AssemblyInfo.cpp" />
+    <ClCompile Include="AssemblyInfo.cpp">
+      <RuntimeLibrary Condition="'$(Configuration)|$(Platform)'=='Release|x64'">MultiThreadedDLL</RuntimeLibrary>
```

Comment:
Sure. #Resolved.

---
In reply to: [75561097](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75561097) [](ancestors = 75561097,75560858,75553518,75537238,75174478)